modules/acquia_cms_image/acquia_cms_image.install

```diff
@@ -41,25 +40,135 @@ function acquia_cms_image_module_implements_alter(array &$implementations, strin
  * Update image style to use 'Focal point scale and crop' effect.
  */
 function acquia_cms_image_update_8001() {
-  $module_path = \Drupal::moduleHandler()->getModule('acquia_cms_image')->getPath();
-  $config_optional = $module_path . '/config/optional';
 
-  $source_optional_dir = new FileStorage($config_optional);
-  $config_storage = \Drupal::service('config.storage');
-  // Image styles name for updating Focal point scale and crop effect.
+  // Image styles for updating Focal point scale and crop effect.
   $image_styles = [
-    'image.style.coh_small_square',
-    'image.style.coh_large_landscape',
-    'image.style.coh_small_landscape',
-    'image.style.coh_medium_landscape',
-    'image.style.coh_x_large_landscape',
-    'image.style.coh_xx_small_landscape',
-    'image.style.coh_xx_large_landscape',
-    'image.style.coh_large_super_landscape',
-    'image.style.coh_medium_super_landscape',
-    'image.style.coh_x_large_super_landscape',
+    'image.style.coh_small_square' => [
```

Comment:
@chandan-singh7929 I think we should try by loading image styles and adding effects programatically and this would automatically add effects. Let's try if that works.

```suggestion
foreach ($image_styles as $image_style) {
  $imageStyle = ImageStyle::load($image_style);
  $imageStyle->addImageEffect([
    'id' => 'focal_point_scale_and_crop',
    'weight' => 1,
    'data' => [
      'width' => 568,
      'height' => 568,
      'crop_type' => 'focal_point'
  ]);
  $imageStyle->save();
}
```

Reply:
@vishalkhode1 - width and height for each image style is different, should we keep that in variable and update accordingly?

Reply:
Hi @chandan-singh7929 Then you probably can use it like this:
```suggestion
$image_styles = [
  "image.style.coh_small_square" => [568, 568],
  "image.style.coh_large_landscape" => [1024, 683]
];
foreach($image_styles as $image_style => $size) {
  $imageStyle = ImageStyle::load($image_style);
  list($width, $height) = $size;
  $imageStyle->addImageEffect([
    'id' => 'focal_point_scale_and_crop',
    'weight' => 1,
    'data' => [
      'width' => $width,
      'height' => $height,
      'crop_type' => 'focal_point'
  ]);
  $imageStyle->save();
}
```

Reply:
@vishalkhode1 - This way we have delete the image effect first and then create the add new one, I think updating using config is good option.

Reply:
@chandan-singh7929 This looks good to me. However, I think updating image style configs won't flush the image styles that were already created. Adding image styles programatically would do think as I suggested above. 
Any thoughts on this ?
cc: @mikemadison13

Reply:
@vishalkhode1 - I have update image styles which are getting updated.